### PR TITLE
fix(crux-ui): npm install target script

### PR DESCRIPTION
**Pull request by princefishthrower**
This should solve the issue raised in https://github.com/dyrector-io/dyrectorio/issues/224

### CODE REVIEW

**Comment by polaroi8d** [comment]
[GPG commit signature](https://github.com/dyrector-io/dyrectorio/blob/develop/CONTRIBUTING.md#commit-signatures) missing.

File: web/crux-ui/package.json [hunk 1]
```diff
@@ -24,7 +24,8 @@
     "build": "next build",
     "build:proto": "find proto -name '*.proto' -exec protoc --experimental_allow_proto3_optional --plugin=./node_modules/.bin/protoc-gen-ts_proto --ts_proto_opt=esModuleInterop=true --ts_proto_opt=outputJsonMethods=true --ts_proto_opt=useDate=false --ts_proto_opt=outputServices=grpc-js --ts_proto_out=./src/models/ {} \\;",
     "start": "npm run dev",
-    "test:e2e": "playwright test"
+    "test:e2e": "playwright test",
+    "install:x86": "npm install --target_arch=x64"
```
Comment (polaroi8d) [inline web/crux-ui/package.json:28]: x86 vs x64?

Comment (polaroi8d) [reply web/crux-ui/package.json:28]: I'm missed the Issue title, sorry.

Comment (polaroi8d) [reply web/crux-ui/package.json:28]: Could you please change to `install:x64`?

**Review by robot9706 — APPROVED** [review]
LGTM 😎

**Review by m8vago — APPROVED** [review]

**Pull request merged**